spec/afterbanks_psd2/base_spec.rb

```diff
@@ -1,6 +1,16 @@
 require "spec_helper"
 
 describe AfterbanksPSD2 do
+  let(:test_api_url) {  'https://test.apipsd2.afterbanks.com' }
+
+  around do |example|
```

Comment:
The `around` hook in RSpec is commonly used when working with [metadata](https://rspec.info/documentation/3.4/rspec-core/RSpec/Core/ExampleGroup.html#metadata-class_method) (sorry couldn't find a place with better documentation, RSpec has been changing their docs lastly and the old reliable is gone). For this case, I think it'd suit better just to use before and after;

```ruby
before do
  ENV['AFTERBANKS_API_URL'] = test_api_url
end

after do
  ENV['AFTERBANKS_API_URL'] = nil
end
```

Reply:
fixed